lua/pl/compat.lua

```diff
@@ -25,9 +27,20 @@ end
 function compat.execute (cmd)
     local res1,_,res3 = os.execute(cmd)
     if compat.lua51 and not compat.jit52 then
-        return res1==0,res1
+        if compat.isWindows then
+            res1 = res1 > 255 and res1 % 256 or res1
+            return res1==0,res1
+        else
+            res1 = res1 > 255 and res1 / 256 or res1
```

Comment:
ok clear, but still slightly uncomfortable with LuaJIT with the 52 compat flag not being in the test matrix. I'm not very good with Travis CI, so dunno how hard it would be to add that.